### ISSUE TITLE
no an issue only a question!

### ISSUE THREAD

**Issue by Robiv8** [Veldrovive/MMM-Page-Selector#3]
Hallo,
can you please publish a config.js example.
me and many other newbies have difficulties implementing this.

Sorry for posting here!

**Comment by Veldrovive** [comment]
No problem. Is there anything specific you want? Certain modules or behavior?

**Comment by Robiv8** [comment]
Thanks for answer,
Nothing special, but the config.js of your picture example would be ideal for getting started.
For a beginner, this is a bit confusing with the short example from the description.
Thank you

**Comment by Veldrovive** [comment]
I don't have my original config anymore, but here is a more up to date one with my favorite modules.
![Imgur](https://i.imgur.com/1GG9wht.gif)
```
var config = {
	address: "localhost", 
	port: 8081,
	ipWhitelist: ["127.0.0.1", "::ffff:127.0.0.1", "::1"],

	language: "en",
	timeFormat: 24,
	units: "metric",

	modules: [
		{
			"module": "MMM-Page-Selector",
			"position": "top_center",
			"config": {
				"defaultPage": "main",
				"displayTitle": true,
				"selectPageNotif": [],
				"incrementPageNotif": [],
				"decrementPageNotif": []
			}
		},
		{
			"module": "alert",
			"pages": "all",
		},
		{
			"module": "MMM-Bob-Ross",
			"position": "top_bar",
			"pages": {"main": "bottom_left"},
			"config": {
				
			}
		},
		{
			"module": "clock",
			"position": "top_right",
			"pages": {"main": "top_right", "fun": "top_left"}
		},
		{
			"module": "weatherforecast",
			"position": "bottom_center",
			"pages": {"main": "bottom_right"},
			"config": {
				"locationID": "REMOVED",
				"appid": "REMOVED"
			}
		},
		{
			"module": "calendar",
			"header": "US Holidays",
			"position": "top_right",
			"pages": {"main": "top_left"},
			"config": {
				"calendars": [
					{
						"symbol": "calendar-check-o ",
						"url": "webcal://www.calendarlabs.com/templates/ical/US-Holidays.ics"
					}
				]
			}
		},
		{
			"module": "MMM-page-indicator",
			"position": "bottom_center",
			"pages": "all",
		},
		{
			"module": "MMM-Lunartic",
			"position": "bottom_right",
			"pages": {"fun": "bottom_right"},
		},
		{
			"module": "MMM-Astronauts",
			"position": "bottom_left",
			"pages": {"fun": "bottom_left"},
		},
		{
			"module": "MMM-Reddit",
			"position": "bottom_right",
			"pages": {"fun": "top_right"},
			"config": {
				"subreddit": "earthporn",
				"displayType": "image",
				"imageQuality": "high",
				"count": 10,
				"show": 1,
				"width": 500,
				"showAll": true,
				"maxImageHeight": 300
			}
		},
		{
			"module": "MMM-Voice-Commands",
			"pages": "all",
			"config": {
				debug: true,
				"autoStart": true,
				"commands": {
					"Play video": "ROSS_PLAY_VIDEO",
					"Pause video": "ROSS_PAUSE_VIDEO",
					"Show picture": "ROSS_SHOW_IMAGE",
					"Unpause video": "ROSS_UNPAUSE_VIDEO",
					"New Picture": "ROSS_NEW_IMAGE",
					"test": function(){
						alert("Heard");
					}
				}
			}
		},
		{
			"module": "MMM-Cursor",
		}
	]

};
```
This one is a bit long and complicated so if you have any questions about it feel free to ask.

**Comment by Robiv8** [comment]
woow
Many Thanks for this! Amazing
Now I can start to study how to build it up :-)

**Comment by Robiv8** [comment]
Thans, yes I have 1 question.

as an an example

"module": "calendar",
"header": "US Holidays",
"position": "top_right",
"pages": {"main": "top_left"},
"config": {
"calendars":
...

I do not understand why "position": top_right but in "pages" then top_left

**Comment by Veldrovive** [comment]
Ah, this can be a point of confusion. It stems from the fact that this module doesn't override magic mirror's method of rendering modules, it just extends it. In order for the module to be shown, a position prop must be set, but since page-selector moves and sets visibility before any modules actually show up on the screen, it doesn't matter what position is set to, only what position it is in on the specified pages.
In the case of the calendar module, when the mirror first starts, the magic mirror will place the module into the `top_right` position, but before you as the user see it, page-selector will move it into the `top_left` position since `main` is the default page for startup.

In summary, the `"position"` prop can be anything as long as it is there. What really matters is the `"pages"` prop and where the module is positioned inside that. 
The only exception is if `"pages"` is set to `"All"` in which case the module will always appear in the position set by the `"position"` prop.

**Comment by Robiv8** [comment]
Thank YOU very much for the explain.
Now i understand :-)

**Comment by DanielSchacht129** [comment]
hey i have a small question,  i have installed MMM-Voice-Commands, MMM-Page-Selector and MMM-page-indicator. But it seems like they dont work properly, do i have to install any aditional voice libraries ?

**Comment by Veldrovive** [comment]
Ah, I'm almost certain I know what the issue is.
MMM-Voice-Commands is built off of annyang which doesn't work properly in electron so magic mirror has to be run in serveronly mode for the module to function correctly. I thought I had updated the readme to reflect that, but I guess not. I will do so now.
If that is not the issue then posting a log of error messages from the console would go a long way to us figuring out the issue.

**Comment by DanielSchacht129** [comment]
Thank you very much, now its working 👍

**Comment by DanielSchacht129** [comment]
I have another problem. On all other pages except my main page my own css rules are not applied, is there a solution to my problem ?

**Comment by Veldrovive** [comment]
Do you mean that you have a `custom.css` file that only works on the default page?
If so, that is a very odd problem indeed. It may be a quirk of the underlying interaction between this module and magic mirror, or it may just be something small that can just be fixed through the config.
Either way, it would be useful to have both your `config.js` as well as your `custom.css` so I can see exactly what is going on.

**Comment by DanielSchacht129** [comment]
So the problem is, if I move the modules adapted in the css file to the home page, then the adaptations will work, but on all other pages unfortunately not anymore. I sent you my code by mail

**Comment by Veldrovive** [comment]
Ah, I'm glad you solved your problem. CSS can be quite finicky.

**Comment by AchimKalthoff** [comment]
Please Help: On the first page I have all my standard modules, on the second page I habe only MMM-WordClock. But it dosen't work. Is there an problem with IFrame?

**Comment by Veldrovive** [comment]
Hey,
If you would, can you put this in a new issue? This is unrelated to the original topic.

I haven't run into any issues with IFrames. In order to debug this, I would need your `config.js` file.

**Comment by MoonHyunSeong** [comment]
I need your help. 
Please refer to the fact that I used a translator.

We made several pages using the page-selector and want to move pages through voice recognition.

But there is a problem. 
1. I don't know what server-only mode is. 
And how to do it.

2. I use voice-commands, but I don't recognize my voice. As a result of testing on Annyang site, there is no problem with my microphone.
